support/cas-server-support-oidc-core/src/main/java/org/apereo/cas/oidc/OidcConstants.java

```diff
@@ -51,6 +51,10 @@
      * The access token hash.
      */
     String CLAIM_AT_HASH = "at_hash";
+    /**
+     * The session identifier claim.
+     */
+    String CLAIM_SESSIOND_ID = "sid";
```

Comment:
```suggestion
    String SLO_SESSIOND_ID = "sid";
```

Reply:
I'd prefer to keep the "CLAIM" tag in the name...